types/stripe/index.d.ts

```diff
@@ -737,12 +738,17 @@ declare namespace Stripe {
              * charge if a partial refund was issued), positive integer or zero.
              */
             amount_refunded: number;
+            
+            /**
+             * ID of the Connect application that created the charge. [Expandable]
+             */
+            application?: string | null;
```

Comment:
This property is `Expandable` which means it can be expanded into an object (see `source_transfer` on L894 as an example). Can you also add the object interface to the type (creating the interface if need be)?